finagle-netty4/src/main/scala/com/twitter/finagle/netty4/ssl/server/Netty4ServerSslHandler.scala

```diff
@@ -74,6 +144,12 @@ private[netty4] class Netty4ServerSslHandler(params: Stack.Params)
     pipeline.addFirst("ssl", sslHandler)
   }
 
+  private[this] def addSniHandler(channel: Channel, factory: SslServerEngineFactory): Unit = {
```

Comment:
private[this] def addSniHandler(
      channel: Channel,
      factory: SslServerEngineFactory,
      sniMapping: Map[String, SslServerConfiguration]
    ): Unit = {
      ...
    }